src/services/transactions_list.rs

```diff
@@ -50,20 +50,20 @@ pub async fn get_all_transactions(
             .as_ref()
             .and_then(|link| extract_query_string(link))
             .map(|link| {
-                context.build_absolute_url(uri!(
-                    crate::routes::transactions::all: safe_address,
+                context.build_absolute_url(uri!(crate::routes::transactions::all(
```

Comment:
this now looks more like the method actually being called than the lambda. There is no change in behaviour between this and the previous code?

Reply:
For us it only changed the syntax and I like the new syntax more :D ... but there are also more changes, would have to search for the commit where they changed it to find the details :/